### DISCUSSION TITLE
UNBLOCK SOME WEBS

### DISCUSSION THREAD

**Discussion by CarmenTT** [SelfControlApp/selfcontrol#789] [Q&A]
Hello!

I used this app for the mac of my kids and after using it for about a month I decided to extend the time for about 65 days. The problem is that the app blocked google.com and the kid needs it to do all her homework. I did not block that webpage or add it so I am in a big problem, and I guess some of you have been in the same situation. I just send an email to the creator but it can take time which I understand.

I try to download the Fireball because I read it would not work there but still, then I am checking to buy a VPN, do you have any recommendation that can help me solve this problem.

Thank you!

**Comment by gabrielf11** [comment]
By now it's probably unblocked, but if you ever have this problem again try reading this article: http://www.joshuakehn.com/2010/10/25/Defeating-SelfControl.html
